homeassistant/components/unifi/strings.json

```diff
@@ -46,15 +46,15 @@
           "dpi_restrictions": "Allow control of DPI restriction groups"
         },
         "description": "Configure client controls\n\nCreate switches for serial numbers you want to control network access for.",
-        "title": "UniFi options 2/3"
+        "title": "UniFi Network options 2/3"
```

Comment:
Make sure to update the other steps as well. Since this simple thing was missed I suggest going through everything a bit more focused :)

Reply:
Oh crap. I did miss the others!

Reply:
I started to quickly go through the integration and found this https://github.com/home-assistant/core/blob/20fbb5b95136b6f089e2fa04198c56d2b7f224d6/homeassistant/components/unifi/__init__.py#L68

Please review all files again.

Reply:
To be fair, I did the first draft of this at like 1 AM. I always forget to re-review the code the day after.

Reply:
My comment was in reaction to you fixing the other two steps but didn't review the rest as my previous comment suggested. It's fine to iterate changes.

Reply:
Sorry, I have a lot going on around the house today. I will definitely give everything a second look once I can.

Reply:
Okay, now everything should look better.